src/main/java/lotto/constant/exception/ErrorMessage.java

```diff
@@ -0,0 +1,44 @@
+package lotto.constant.exception;
+
+import java.text.DecimalFormat;
+import lotto.constant.LottoConstant;
+
+/**
+ * 프로젝트에서 사용하는 에러 메시지.
+ *
+ * <p>{@code build} 메서드로 메시지를 생성할 수 있다.
+ *
+ * <p>각 메시지에 대한 설명은 그 자체로 명확하므로 주석을 생략한다.
+ */
+
+public enum ErrorMessage {
+    LOTTO_NUMBER_COUNT_INVALID("로또 번호는 " + LottoConstant.LOTTO_NUMBER_SIZE + "개여야 합니다: "),
+    LOTTO_NUMBER_DUPLICATED("로또 번호에 중복이 존재합니다: "),
+    LOTTO_NUMBER_OUT_OF_RANGE("로또 번호의 숫자 범위는 " + LottoConstant.LOTTO_NUMBER_LOWER_RANGE_INCLUSIVE + "~"
+            + LottoConstant.LOTTO_NUMBER_UPPER_RANGE_INCLUSIVE + "까지입니다: "),
+    PURCHASE_AMOUNT_IS_NOT_DIVISIBLE_BY_LOTTO_PRICE("구입 금액은 "
+            + LottoConstant.LOTTO_PRICE_FORMATTED_TEXT + "원 단위여야 합니다: "),
+    PURCHASE_AMOUNT_IS_NOT_POSITIVE("구입 금액은 양수여야 합니다: "),
+    DUPLICATES_BETWEEN_WINNING_NUMBERS_AND_BONUS_NUMBER("보너스 번호는 당첨 번호와 중복될 수 없습니다: "),
+    OUT_OF_RANGE_MATCH_COUNT("맞춘 당첨 번호의 수가 유효한 범위를 벗어났습니다: "),
+    ;
```

Comment:
현재 예외 상수들이 모두 연속해서 붙어 있는데 서로 다른 맥락의 예외들(예: 로또 번호 검증 / 구입 금액 검증 / 입력 형식 검증 등) 사이에 적절히 개행을 넣어 그룹화하면 가독성이 더 좋아질 것 같아요~
```suggestion
    LOTTO_NUMBER_COUNT_INVALID("로또 번호는 " + LottoConstant.LOTTO_NUMBER_SIZE + "개여야 합니다: "),
    LOTTO_NUMBER_DUPLICATED("로또 번호에 중복이 존재합니다: "),
    LOTTO_NUMBER_OUT_OF_RANGE("로또 번호의 숫자 범위는 " + LottoConstant.LOTTO_NUMBER_LOWER_RANGE_INCLUSIVE + "~"
            + LottoConstant.LOTTO_NUMBER_UPPER_RANGE_INCLUSIVE + "까지입니다: "),
            
    PURCHASE_AMOUNT_IS_NOT_DIVISIBLE_BY_LOTTO_PRICE("구입 금액은 "
            + LottoConstant.LOTTO_PRICE_FORMATTED_TEXT + "원 단위여야 합니다: "),
    PURCHASE_AMOUNT_IS_NOT_POSITIVE("구입 금액은 양수여야 합니다: "),
    
    DUPLICATES_BETWEEN_WINNING_NUMBERS_AND_BONUS_NUMBER("보너스 번호는 당첨 번호와 중복될 수 없습니다: "),
    
    OUT_OF_RANGE_MATCH_COUNT("맞춘 당첨 번호의 수가 유효한 범위를 벗어났습니다: "),
    ;
```

Reply:
> 현재 예외 상수들이 모두 연속해서 붙어 있는데 서로 다른 맥락의 예외들(예: 로또 번호 검증 / 구입 금액 검증 / 입력 형식 검증 등) 사이에 적절히 개행을 넣어 그룹화하면 가독성이 더 좋아질 것 같아요~

나름 논리적 순서대로 배치는 했는데, 개행까지는 신경쓰지 못했네요. 말씀하신 부분을 적용하면 리뷰어들이 읽기 더 편해지겠어요!

소중한 조언 감사해요.